### PR TITLE
emptvols -> volumes=yes

### PR DESCRIPTION
**Pull request by ashetty1**

### CODE REVIEW

**Comment by cdrage** [comment]
LGTM

**Comment by surajnarwade** [comment]
LGTM :)

File: script/test_in_openshift/lib.sh [hunk 1]
```diff
@@ -192,7 +192,7 @@ function convert::kompose_up () {
     # Usage: convert::kompose_up <docker_compose_file>
     local compose_file=$1
     convert::print_msg "Running kompose up ..."
-    ./kompose up --provider=openshift --emptyvols -f $compose_file
+    ./kompose up --provider=openshift --volumes empty -f $compose_file
```
Comment (kadel) [inline script/test_in_openshift/lib.sh]: This is suspicious. 
Isn't correct value `emptyDir`? It looks like there is no validation for `--volumes` values.
ping @cdrage

Comment (kadel) [reply script/test_in_openshift/lib.sh]: created https://github.com/kubernetes/kompose/issues/814

Comment (ashetty1) [reply script/test_in_openshift/lib.sh]: @kadel looking into it. You are right. Also, the help text needs to change,

**Pull request merged**